nss-day-cohort-41/react-nutshell-the-macadamias
# News start

Description
This is the News Feed section of Nutshell

Type of change

New feature : Displays news articles of Active user and users friends. Allows active user to create a new article and delete active user article

Testing instructions:
You will first need to install "React Tiny Link"

```npm install --save react-tiny-link```

Delete and Post new functionality in place. 

Delete will show a modal to confirm the delete

Post new article will open the input fields. The discard button will close the fields and reset state. Submit button will POST new article. If any fields are left blank, an alert modal will open preventing submit from moving forward.


Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
